website_multi_image/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Multiple images base
+====================
+
+This module extends the functionality of ``base_multi_image`` and provides some
+technical tools useful to combine it with ``website``.
+
+Basically, it provides reusable templates for objects owning multiple images.
+
+Installation
+============
+
+This module adds no visible behavior, just some reusable tools for developers.
+It will be automatically installed if any other module you have needs those
+tools, but otherwise you probably don not need it.
+
+Development
+===========
+
+To develop a module based on this one:
+
+* See module ``website_event_multi_image`` as an example.
+* Use the provided templates in frontend website views.
```

Comment:
I think this sentence should be rewritten to:
```
Inherit from the provided QWeb templates for creating website views.
```
And then you can list the current views IDs with a little explanation of each.